iphone/TitaniumKit/TitaniumKit/Sources/API/TiWindowProxy.m

```diff
@@ -558,6 +558,13 @@ - (void)openOnUIThread:(NSArray *)args
           [theController setModalPresentationStyle:style];
         }
       }
+
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
```

Comment:
Use the new `IS_IOS_SDK_13` constant

Reply:
This macro will not work inside framework.

Reply:
Duh, of course.